test/Interop/Cxx/operators/member-inline-module-interface.swift

```diff
@@ -1,6 +1,7 @@
 // RUN: %target-swift-ide-test -print-module -module-to-print=MemberInline -I %S/Inputs -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
 
 // CHECK: struct LoadableIntWrapper {
```

Comment:
Can you please add a test with a unary friend operator? e.g. `friend LoadableIntWrapper operator-(const LoadableIntWrapper& obj)`